### ISSUE TITLE
Comment text in attribute 'obsolete' is not displayed in the generated client

### ISSUE THREAD

**Issue by Sfinks7s1** [RicoSuter/NSwag#2595]
Hello guys,

In my controller method marked as [Obsolete("Reason")] but in generated client for it 'obsolete' attribute without comment - [Obsolete]
Is there a way around this problem?

In .nswag config  "ignoreObsoleteProperties": false

**Comment by RicoSuter** [comment]
OpenAPI does not have the means to store the obsolete message in the spec... that's why we cant generate it.

**Comment by RicoSuter** [comment]
We could add it as x-obsoleteMessage or something and generate it in the clients.

**Comment by nistre** [comment]
I'm not sure what the x-obsoleteMessage is but any way of including the Obsolete Message I think would be great.

**Comment by bricelam** [comment]
You could also just append it to the description or summary.

> This operation is obsolete: {message}

**Comment by nistre** [comment]
Of course, but there is still no warning integration impacting expected functionality of properly informing consumers.

**Comment by bricelam** [comment]
Oh I see, I was only thinking about getting it into the swagger.json file, but you want to round-trip it back into generated clients.
